Telecomm/telecommServer.py

```diff
@@ -32,6 +32,25 @@
 
 #TODO
 #Document input/output values
+#Instructions for usage:
+#   1) In the registry, create the directory /Servers/Telecomm
+#   2) Create a key named 'smsUsers' with value
+#       [("USERNAME1","TENDIGITPHONENUMBER@txt.att.net"),...
+#       ,("USERNAMEM","TENDIGITPHONENUMBER@vtext.com")]
+#      Note that the domain name depends on the user's
+#      mobile phone provider. "USERNAME" must be ALL caps.
+#   3) Create a key named 'domain' with a value that
+#      matches the domain name of the email address
+#      from which you wish to send messages.
+#      e.g. domain = "physics.someschool.edu"
+#   4) Create a key named 'smtpServer' whose value is
+#      the smtp server you wish to use.
+#   5) (Optional) Create a key named 'password'
```

Comment:
> If the concern is managing permissions, then would storing the password in a text or yaml file really improve things?

Yes. Presumably your computers have access control.

> Do you guys have a single machine dedicated to using this server?

No, but that's not really the point. The point is to not use LabRAD as an access control system, because it's not (yet) designed for that (although @maffoo has been making great strides in that direction).

It's better to distribute the passwords some other way; whether by talking to each other or using some other software tool is irrelevant as long as it's as secure. This leaves the question of how to get the telecomm server to find out about this password.

We recently had to deal with a very similar situation regarding access to an SQL database through some scripts in our own code repo. My first solution was exactly what you're suggesting: put a password file in the same repo but add it to `.gitignore` so it doesn't wind up in version control. Again, how you tell other what password to put in that file is entirely up to you. However, that was a bit limiting and kind of dangerous, so now we have users put the passwords in a file in their home directory, and our scripts simply look there. You can use `os.path.expanduser` to get the home directory on Mac/Linux/Windows.